docs/src/main/asciidoc/jreleaser.adoc

```diff
@@ -0,0 +1,594 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Packaging And Releasing With JReleaser
+
+include::./attributes.adoc[]
+:jreleaser-version: 0.9.1
+
+:toc: macro
+:toclevels: 4
+:doctype: book
+:icons: font
+:docinfo1:
```

Comment:
Can you remove all this? It's managed automatically in the website and I don't want it to conflict.

Reply:
Sure. I used `getting-started.adoc` as a starting point which also defines these attributes btw ;-)